<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContGeneral</class>
 <widget class="QMainWindow" name="ContGeneral">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="windowTitle">
   <string>Catalogo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ContablesPrincipal.qrc">
    <normaloff>:/iconos/addressbook.png</normaloff>:/iconos/addressbook.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Saldo Deudor</string>
    </property>
    <widget class="QLineEdit" name="saldoDeudor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Descripci√≥n</string>
       </property>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>261</width>
          <height>151</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>530</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>540</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Categoria</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Cuenta</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Haber</string>
       </property>
       <widget class="QLineEdit" name="haber">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>121</width>
          <height>25</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Debe</string>
       </property>
       <widget class="QLineEdit" name="debe">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>22</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>308</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Saldo Acreedor</string>
    </property>
    <widget class="QLineEdit" name="saldoAcreedor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>279</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Nombre de la Cuenta</string>
    </property>
    <widget class="QLineEdit" name="codigoCuenta">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="transaccion">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>540</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Transaccion</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="Catalogo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>501</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>375</width>
      <height>680</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>Catalogo de Cuentas</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>46</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../ContablesPrincipal.qrc">:/iconos/kedit.png</pixmap>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>400</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;El catalogo de cuentas le permite observar los diferentes saldos en las cuentas, ademas puede hacer asientos contables en &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Crear Transaccion. &lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Ademas puede crear cuentas y categorias, para conglomerar subcuentas.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="salirMain">
   <property name="icon">
    <iconset>
     <normaloff>iconos/exit.png</normaloff>iconos/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Salir</string>
   </property>
  </action>
  <action name="action_Ayuda_Rapida">
   <property name="icon">
    <iconset>
     <normaloff>iconos/help.png</normaloff>iconos/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Ayuda Rapida</string>
   </property>
  </action>
  <action name="action_Acerca_de_Sic2008">
   <property name="icon">
    <iconset>
     <normaloff>iconos/about_kde.png</normaloff>iconos/about_kde.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Acerca de Sic2008</string>
   </property>
  </action>
  <action name="action_Eliminar">
   <property name="icon">
    <iconset>
     <normaloff>iconos/editdelete.png</normaloff>iconos/editdelete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Eliminar</string>
   </property>
  </action>
  <action name="actionCargar">
   <property name="text">
    <string>Cargar</string>
   </property>
  </action>
  <action name="actionAbonar">
   <property name="text">
    <string>Abonar</string>
   </property>
  </action>
  <action name="action_Activo">
   <property name="text">
    <string>&amp;Activo</string>
   </property>
  </action>
  <action name="action_Pasivo">
   <property name="text">
    <string>&amp;Pasivo</string>
   </property>
  </action>
  <action name="action_Capital">
   <property name="text">
    <string>&amp;Capital</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../ContablesPrincipal.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salirMain</sender>
   <signal>activated()</signal>
   <receiver>ContGeneral</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>ContGeneral</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
